css/index.css

```diff
@@ -1,85 +1,163 @@
 }
 
 .profile-photo {
     width: 150px;
     height: 180px;
-    margin-top: 20px;
+    margin-top: 100px;/* adjusted margin from 20px*/
     margin-right: 780px;
     border: 2px solid #2cbdedf1;
-    } /* picture added*/
+} /* picture added */
 
 #Experience ul {
     list-style-type: none;
-    padding: 0px;
-    margin: 0px
-     }
-#Experience li{
-    display: block; 
+    padding: 0;
+    margin: 0;
+}
+
+#Experience li {
+    display: flex;               /*flex added */
+    flex-direction:row;
     background-color: #f0f0f0;
-    border: 1px solid #ddd; 
-    border-radius: 8px; 
-    padding: 15px; 
+    border: 1px solid #ddd;
+    border-radius: 8px;
+    padding: 15px;
     margin-bottom: 10px;
-    text-align: center; 
-    }
+
+}
+.experience-item{
+    display:flex;
+    flex-direction:row;
+    flex-wrap: wrap;
+    width: 100%;
+}
+.job, .time {
+    flex: 1;
+    padding: 10px;
+    box-sizing: border-box;
+}
+.job{
+    text-align: right;
+}
+.time{
+    text-align: left;
+}
+
+#Experience li span{
+margin-left: auto; /*added margins*/
+}
 #Experience li:hover {
-    background-color: #e0e0e0;}
-
- /* Transform the style of the "Connect" links */
-    #Connect a {
-        color: #007bff;
-        padding: 10px 15px; 
-        border-radius: 5px; 
-        text-decoration: none;
-        transition: background-color 0.3s ease, color 0.3s ease; /* Smooth transition */
-    }
-    #Connect a:hover {
-        text-decoration: underline;
-    }
-    ul {
-        list-style-type: none;
-        padding: 0;
-        margin: 0; 
-        }
-    
- footer {
+    background-color: #e0e0e0;
+}
+#Experience li div{  /*div added to make it nicer*/
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+
+/* Transform the style of the "Connect" links */
+#Connect ul{
+    display: flex;
+    justify-content: center;
+    gap: 1headepx;
+    padding: 0;
+    margin: 0;
+}
+#Connect a {
+    color: #007bff;
+    padding: 10px 15px;
+    border-radius: 5px;
+    text-decoration: none;
+    transition: background-color 0.3s ease, color 0.3s ease; /* Smooth transition */
+}
+
+#Connect a:hover {
+    text-decoration: underline;
+}
+
+ul {
+    list-style-type: none;
+    padding: 0;
+    margin: 0;
+}
+
+.github, .email, .facebook {
+    max-width: 40%;
+}
+
+footer {
     padding: 10px;
     background-color: #400153f1;
     color: #fff;
     margin-top: 20px;
     text-align: center;
 }
-
+/* header fixed position */
+header {
+    position: fixed;
+    top: 0;
+    width: 100%;
+    max-width: 800px; /* Align with sections */
+    background-color: white;
+    border-radius: 8px;
+    margin: 0 auto; /* Center align */
+    padding: 10px;
+    box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
+    z-index: 1000; /* Ensures the header stays on top of other elements */
+    
+    display:flex;
+    flex-direction: column;
+    align-items:center;
+    }
+    /*fixing navigation bar*/
+    .navigation-bar ul{
```

Comment:
Use the formatter to fix formatting, indentation is not lined up - makes it hard to review.